ldb/export.py

```diff
@@ -294,7 +295,7 @@ def getname(obj):
                     name = obj.get("organization__name_prefix", "")
                     name += " "
                     name += obj.get("organization__name")
-                    name = re.sub("\s+", " ", name)
+                    name = re.sub(" +", " ", name)
```

Comment:
Why is this changed? `\s` matches more than just spaces.

Reply:
Flake8 says:
```
./ldb/export.py:298:36: W605 invalid escape sequence '\s'
./ldb/export.py:320:36: W605 invalid escape sequence '\s'
```

Reply:
Hm it's not a escape sequence but part of the regex, maybe it should be `\\s+` instead?

Reply:
I think it might need to be `r"\s+"`